Poker.BE/Poker.BE.Domain/Game/Hand.cs

```diff
@@ -33,8 +33,10 @@ public class Hand
         #region Properties
         public bool Active { get; set; }
         public Round CurrentRound { get; private set; }
-        public Card[] CommunityCards { get { return communityCards; } }
+        public Pot Pot { get { return pot; } set { pot = value; } }
+        public Card[] CommunityCards { get { return communityCards; } set { communityCards = value; } }
```

Comment:
actually this was just for the EndTest Method in the HandTest class...
It made my life much easier testing it this way... I know it's not optimal but it's not so bad either